packages/webdriverio/src/commands/element/click.js

```diff
@@ -63,10 +63,10 @@
  * @alias element.click
  * @uses protocol/element, protocol/elementIdClick, protocol/performActions, protocol/positionClick
  * @type action
- * @param {object=} options Object (optional)
+ * @param {ClickOptions=}     options        click options (optional)
```

Comment:
is there any way to make it read the object as optional because else the user will always have to pass an empty object or the user will get an error saying `Expected 1 arguments, but got 0.`

Reply:
@erwinheitzman by having a `=` at the end it will be marked as optional argument.